src/server/req_manager.c

```diff
@@ -3364,6 +3418,59 @@ setup_ping(int delay)
 
 /**
  * @brief
+ *		mgr_sched_create - process request to create a sched
+ *
+ *		Creates sched
+ *
+ *  @param[in]	preq	- Pointer to a batch request structure
+ */
+static void
+mgr_sched_create(preq)
+struct batch_request *preq;
+{
+	int bad;
+	char *badattr;
+	svrattrl *plist;
+	pbs_sched *psched;
+	int rc;
+
+	if (strlen(preq->rq_ind.rq_manager.rq_objname) > PBS_MAXSCHEDNAME) {
+		req_reject(PBSE_SCHED_NAME_BIG, 0, preq);
+		return;
+	}
+	if (find_scheduler(preq->rq_ind.rq_manager.rq_objname)) {
+		req_reject(PBSE_SCHEDEXIST, 0, preq);
+		return;
+	}
+
+	psched = sched_alloc(preq->rq_ind.rq_manager.rq_objname);
+
+	/* set the scheduler attributes */
+
+	plist = (svrattrl *) GET_NEXT(preq->rq_ind.rq_manager.rq_attr);
+	rc = mgr_set_attr(psched->sch_attr, sched_attr_def, SCHED_ATR_LAST, plist,
+			preq->rq_perm, &bad, (void *) psched, ATR_ACTION_NEW);
+	if (rc != 0) {
+		reply_badattr(rc, bad, plist, preq);
+		sched_free(psched);
+		psched = NULL;
+	} else {
+
+		/* save the attributes to disk */
+		(void) sched_save_db(psched, SVR_SAVE_FULL);
+		set_sched_default(psched);
+		(void) sprintf(log_buffer, msg_manager, msg_man_set, preq->rq_user,
+				preq->rq_host);
+		log_event(PBSEVENT_ADMIN, PBS_EVENTCLASS_SCHED, LOG_INFO,
+				msg_daemonname, log_buffer);
+		mgr_log_attr(msg_man_set, plist, PBS_EVENTCLASS_SCHED, msg_daemonname,
+		NULL);
```

Comment:
When the coding standard went in with tabs being our standard indent, the 80 character line standard was no longer needed.  I personally find a longer line easier to read than one broken at 80 characters in certain cases.  This is one of them.  The one line form will be 86 characters.  I have no problem dragging my vi windows open a little wider in this case.